src/screens/IndexScreen.js

```diff
@@ -394,12 +417,25 @@ export const IndexScreen = ({ navigation, route }) => {
                         <>
                           <DropdownHeader
                             {...{
+                              data: initialNewsItemsFetch && loading ? {} : data,
                               query,
                               queryVariables,
-                              data: initialNewsItemsFetch && loading ? {} : data,
                               updateListData: updateListDataByDropdown
                             }}
                           />
+
+                          {query === QUERY_TYPES.EVENT_RECORDS && showEventLocationFilter && (
+                            <DropdownHeader
+                              {...{
+                                data: initialNewsItemsFetch && loading ? {} : data,
```

Comment:
how are news items related here?

Reply:
actually the filter here has nothing to do with newsItem. I just forgot it that way. I'll fix it. 👍

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/807/commits/167609db8f6e3f2c14dcb590081f5a57abfa2985